core/tests/protractor/subscriptions.js

```diff
@@ -50,13 +53,9 @@ describe('Subscriptions functionality', function() {
     browser.get('/profile/creator2subscriptions');
     browser.waitForAngular();
     element(by.css('.protractor-test-subscription-button')).click();
-    browser.get(general.USER_PREFERENCES_URL);
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).first().getText()).toMatch(
-      'creator...');
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).last().getText()).toMatch(
-      'creator...');
+    preferencesPage.get();
+    preferencesPage.expectFirstSubscriptionToBe('creator1subscriptions');
```

Comment:
Rename to `expectFirstSubscriptionToMatch`.

Reply:
It's a defined function. I don't think it should be renamed. Still, I did so. Lets see.